src/interfaces/Course.ts

```
@@ -0,0 +1,133 @@
+        _id: Types.ObjectId;
+    }[];
+    rating?: number;
+    purchased?: number;
+    isPublished: boolean;
+    isFree: boolean;
+    category: Types.ObjectId;
+    subCategory: Types.ObjectId;
+    overview: string;
+    topics: string[];
+    duration: {
+        type: Number;
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)